continue-reading.html

```diff
@@ -4,10 +4,141 @@
+                  </p>
+                    <h3 class="totally"><strong>Lorem ipsum dolor, sit amet!</strong><br>
+                    </h3>
+                    <p class="totally_para">
+
+                    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus tenetur consectetur consequatur reiciendis praesentium. Assumenda veritatis omnis magni veniam voluptatum inventore consequatur id impedit, nostrum dolorem vel at enim saepe.Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus tenetur consectetur consequatur reiciendis praesentium. Assumenda veritatis omnis magni veniam voluptatum inventore consequatur id impedit, nostrum dolorem vel at enim saepe.<br><br>
+                  
+                    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus tenetur consectetur consequatur reiciendis praesentium. Assumenda veritatis omnis magni veniam voluptatum inventore consequatur id impedit, nostrum dolorem vel at enim saepe.Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus tenetur consectetur consequatur reiciendis praesentium. Assumenda veritatis omnis magni veniam voluptatum inventore consequatur id impedit, nostrum dolorem vel at enim saepe.Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                    </p>
+                    <h3 class="totally"><strong>Lorem ipsum dolor, sit amet!</strong><br>
+                    </h3>
+                    <p class="totally_para">
+                  
+                  Lorem ipsum dolor sit amet, consectetur adipisicing elit. Possimus officia, molestias recusandae quisquam eveniet odio tenetur atque et laborum? Laborum sit suscipit officia repellat quis eius veritatis reprehenderit ullam est! Lorem ipsum dolor sit amet consectetur adipisicing elit. Nisi eos voluptatibus aut deleniti harum similique necessitatibus excepturi, voluptas nesciunt et? Animi eaque, dolorem eius provident quis magnam quas. Repellendus, veniam?
+                </p>
+                </div>
+          </div>
+  
+      
+          
+  
+          <!-- RIGHT SIDE POSTS BELOW -->
+  
+          <div id="right_container">
+              <div id="another_post">
+                  <div id="another_post_date" class="date">July 30</div>
+                  <h3 class="post_heading"><a href="blog-post.html"><strong>Just another post</strong></a></h3>

```

Comment:
Having the title as a link is a good way to do this, but you might want to have it look like a link. Another option could be to wrap the whole text content in the `<a>` tag so that if you click anywhere on the preview of the post it will bring you to the post itself.

This goes for all of these style sections

Reply:
Ok! Yeah, styling it this way was not my preference - I just followed the project video walkthrough as closely as possible (which had it change color on hover), though I agree that it would make more sense if the whole post was clickable. I'll go ahead and add that functionality!